### PR TITLE
79 log models to neptunes registry

### PR DESCRIPTION
**Pull request by wtaisner**
I also added CPU only version of the docker

### CODE REVIEW

File: DockerfileCPU [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+RUN apt-get clean
+RUN rm -rf /var/lib/apt/lists/*
+
+RUN pip install sklearn
+RUN pip install pandas
+RUN pip install torchmetrics
+RUN pip install addict
+RUN pip install yapf
+RUN pip install neptune-client
+RUN pip install h5py
```
Comment (jkarolczak) [inline DockerfileCPU:21]: you can compress these to a single layer using:
`RUN pip install sklearn pandas torchmetrics addict yapf neptune-client h5py`

This should result in faster building, especially when you rebuild the container.

File: cfg/models/minkloc3dv2.yaml [hunk 2]
```diff
@@ -14,4 +14,5 @@ quantization_step: 0.01
 normalize_embeddings: False
 
 input_dim: 1
-output_dim: 219
+
+output_dim: 3
```
Comment (jkarolczak) [inline cfg/models/minkloc3dv2.yaml:18]: Can we undo this change?

File: cfg/train.yaml [hunk 3]
```diff
@@ -1,12 +1,12 @@
-model: PointNet
-device: cuda
-dataset_dir: ../../data/blobs_full
-dataset_file: ../../data/labels_three.csv
+model: MinkLoc3Dv2
+device: cpu
+dataset_dir: /home/witold/blobs_full
+dataset_file: ../data/labels_three.csv
```
Comment (jkarolczak) [inline cfg/train.yaml:4]: Another one to undo

File: src/models/__init__.py [hunk 4]
```diff
@@ -4,7 +4,7 @@
 
 MODELS = {
     "transloc3d": transloc3d,
-    "minkloc3dv2": minkloc3dv2,
+    "minkloc3d": minkloc3dv2,
```
Comment (jkarolczak) [inline src/models/__init__.py:7]: gif

File: src/log.py [hunk 5]
```diff
@@ -1,4 +1,5 @@
 import os
+import warnings
```
Comment (jkarolczak) [inline src/log.py:2]: Do we need this import? Looks unused to me

**Review by jkarolczak — CHANGES_REQUESTED** [review]

File: DockerfileCPU (hunk 1 above)
Comment (jkarolczak) [reply DockerfileCPU:21]: You can just resolve this comment if you don't want to change the snippet. It's more like a hint or good practice rather than a change request.

Comment (wtaisner) [reply DockerfileCPU:21]: if it wasn't for pasted piece of code I'd have resolved it, but since you did the dirty work, I will honor it and copy paste the code

File: src/log.py (hunk 5 above)
Comment (wtaisner) [reply src/log.py]: gone with the wind

File: cfg/train.yaml (hunk 3 above)
Comment (wtaisner) [reply cfg/train.yaml]: another one bites the dust

**Comment by wtaisner** [comment]
@jkarolczak have fun once again

**Review by jkarolczak — APPROVED** [review]
meme
